docs/design/data-modeling.md

```diff
@@ -159,3 +172,8 @@ This technique enables a simple range query to iterate every object
 with a target role. While you can't yet do prefix queries within
 TigerBeetle, you could do a prefix query in an external database to
 resolve `id`s and pass the resolved `id`s into TigerBeetle.
+
+#### Random Identifier
```

Comment:
Just below the "Examples (Advanced)" section, we still stay Random IDs are recommended for most applications and link here.

(GitHub won't let me comment there :sweat:)